tests/unit/contract_info_test.py

```
@@ -307,3 +326,119 @@ def test_proto_conversion_minimal_fields():
+    
+    contract_info = ContractInfo._from_proto(proto)
+    
+    assert contract_info.contract_id == ContractId(0, 0, 200)
+    assert contract_info.staked_account_id is None
+    assert contract_info.staked_node_id is None
+    assert contract_info.decline_staking_reward is None
+
+
+def test_from_proto_with_staked_node_id():
+    """Test from_proto with staked_node_id (staked to node)"""
+    public_key = PrivateKey.generate_ed25519().public_key()
```

Ruff (0.14.14)

[error] Local variable `public_key` is assigned to but never used

Remove assignment to unused variable `public_key`

(F841)